transport/rif_comms/client.py

```diff
@@ -62,11 +62,10 @@ def is_subscribed_to(self, rsk_address: Address) -> bool:
         :param rsk_address: RSK address which corresponds to the topic which is being checked for subscription
         :return: boolean value indicating whether the client is subscribed or not
         """
-        our_peer_id = self._get_peer_id(self.rsk_address.address)
-        topic_id = self._get_peer_id(to_checksum_address(rsk_address))
+        topic_id = self._get_peer_id(rsk_address)
         return self.stub.HasSubscriber(
             Subscriber(
-                peerId=our_peer_id,
+                peerId=topic_id,
                 channel=Channel(channelId=topic_id)
             )
```

Comment:
i still don't understand this.

i interpret the previous code as:

> check if `our_peer_id` is subscribed to this `topic_id`

where:
- `our_peer_id` is the result of looking up our own RSK address in the DHT
- `topic_id` is the result of looking up the destination RSK address in the DHT

this made sense to me :man_shrugging: 

now, this looks like: 
> check if the destination peer is subscribed to this `channelId` 

because:
- `peerId` is the result of looking up the **recipient** RSK address in the DHT
- `channelId` is also the same as the previous point, which is confusing :thinking: 

can you please explain this further?

Reply:
AFAIK event its redundat to call with the same parameter twice, maybe on other scenarios you want to check if another peer is subscribed to the same channel

on this scenario, is the node behind the client that wants to check if it is subscribed to the peer id of the partner on that same channel

not too happy with this

Reply:
ok